libnavigation-router/src/main/java/com/mapbox/navigation/route/internal/hybrid/HybridRouterHandler.kt

```diff
@@ -0,0 +1,248 @@
+package com.mapbox.navigation.route.internal.hybrid
+
+import com.mapbox.api.directions.v5.models.DirectionsRoute
+import com.mapbox.api.directions.v5.models.RouteOptions
+import com.mapbox.base.common.logger.Logger
+import com.mapbox.base.common.logger.model.Message
+import com.mapbox.base.common.logger.model.Tag
+import com.mapbox.navigation.base.route.RouteRefreshCallback
+import com.mapbox.navigation.base.route.RouteRefreshError
+import com.mapbox.navigation.base.route.Router
+
+internal sealed class RouterHandler(
```

Comment:
This should be name `HybridRouterHandler` (or change the file name to `RouterHandler.kt`) to avoid `ktlint` error https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-android/11272/workflows/d98e1646-d835-4709-b0ad-1f2beff79232/jobs/44945